### PR TITLE
feat(helm): added timeout input for deploy

### PR DESCRIPTION
**Pull request by Wagan8r**

### CODE REVIEW

File: .github/workflows/helm-deploy-eks.yml [hunk 1]
```diff
@@ -58,6 +58,11 @@ on:
         description: 'Environment for deployment'
         required: true
         type: string
+      timeout:
+        description: 'Duration to wait for a successful deployment'
```
Comment (zdbrandt) [inline .github/workflows/helm-deploy-eks.yml:62]: Super minor but should we clarify if this is seconds or milliseconds?

Comment (Wagan8r) [reply .github/workflows/helm-deploy-eks.yml]: Yeah, that's a great idea. Updated.

**Review by zdbrandt — APPROVED** [review]

File: .github/workflows/helm-deploy-eks.yml [hunk 2]
```diff
@@ -58,6 +58,11 @@ on:
         description: 'Environment for deployment'
         required: true
         type: string
+      timeout:
+        description: 'Duration in to wait for a successful deployment e.g. 300s, 10m, 1h'
```
Comment (zdbrandt) [inline .github/workflows/helm-deploy-eks.yml:62]:
```suggestion
        description: 'Duration to wait for a successful deployment e.g. 300s, 10m, 1h'
```

Comment (Wagan8r) [reply .github/workflows/helm-deploy-eks.yml:62]: Thanks for the catch!

**Pull request merged**